I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/sim.rpy", line 12, in script call
    call screen tie
  File "game/control.rpy", line 129, in script
    show peep_nocry with none
  File "game/control.rpy", line 129, in <module>
    show peep_nocry with none
NameError: name 'none' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/sim.rpy", line 12, in script call
    call screen tie
  File "game/control.rpy", line 129, in script
    show peep_nocry with none
  File "D:\PROGRAM\RENPYY\renpy-7.3.5-sdk\renpy\ast.py", line 1382, in execute
    trans = renpy.python.py_eval(self.expr)
  File "D:\PROGRAM\RENPYY\renpy-7.3.5-sdk\renpy\python.py", line 2059, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "D:\PROGRAM\RENPYY\renpy-7.3.5-sdk\renpy\python.py", line 2052, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/control.rpy", line 129, in <module>
    show peep_nocry with none
NameError: name 'none' is not defined

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Pride and Submission 1.3.2
Thu Jul 01 19:23:20 2021
